### DISCUSSION TITLE
Programmatic access to health status

### DISCUSSION THREAD

**Discussion by obrunsmann** [spatie/laravel-health#97] [General]
Any interest of implementing something like

```php
Health::isHealthy();
```

or

```php
Health::getStatus('external_api');
```

Could be useful for 2 cases:

1) Showing a small badge on the main page (healthy/having issues) or adding a hint when downgrades expected.
2) Could check the status before hitting external services

**Comment by freekmurze** [comment]
Yup, that could be nice. Feel free to send a PR. Make sure to include tests and updated docs. 👍
